mail/isync/files/mbsync.plist

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple Computer//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+  <dict>
+    <key>KeepAlive</key>
+    <false/>
+    <key>Label</key>
+    <string>@LABEL@</string>
+    <key>ProgramArguments</key>
+    <array>
+      <string>@PREFIX@/bin/mbsync</string>
+      <string>-a</string>
+    </array>
+    <key>Disabled</key>
+    <true/>
+    <key>StartInterval</key>
+    <integer>300</integer>
```

Comment:
An interval of 5 minutes, while reasonable, won't be good for everyone. For this reason, I think it might be preferable to provide the plist as a reference under the `share/doc/isync/examples` directory. One can then copy it under `~/Library/LaunchAgents` and customize it if needed.

Reply:
Yea that makes sense, although it is nice to have the convenience of `port load isync`. Is there a way to get the best of both worlds (i.e. the convenience and flexibility)?

Since our plist file is not actually part of the isync distribution itself, I suggest installing it to `/opt/local/share/examples/isync` instead of `/opt/local/share/doc/isync/examples/`. The former directory is where e.g. Syncthing installs its plist file.

Reply:
Updated in 4c55ed5

Reply:
> Yea that makes sense, although it is nice to have the convenience of `port load isync`. Is there a way to get the best of both worlds (i.e. the convenience and flexibility)?

I am afraid not.

> Since our plist file is not actually part of the isync distribution itself, I suggest installing it to `/opt/local/share/examples/isync` instead of `/opt/local/share/doc/isync/examples/`. The former directory is where e.g. Syncthing installs its plist file.

Sounds good.